server/server.js

```diff
@@ -1,15 +1,71 @@
+require("dotenv").config();
 const express = require("express");
 const app = express();
 const port = process.env.PORT || 5000;
+const cors = require("cors");
```

Comment:
Not sure which version of Node you are using, but going forward I think it's better if you get acquainted with the `import/export` modules that was introduced in ES6. Here's a comprehensive [guide](https://www.digitalocean.com/community/tutorials/js-modules-es6) on how to start using them and why are they so powerful.

Reply:
Thanks Giorgi for sharing about ES6! My Node version is 18.17.1. I will read it.